dap-etl/transformation/src/main/scala/org/broadinstitute/monster/dap/eols/DeathTransformations.scala

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.monster.dap.eols
+
+import org.broadinstitute.monster.dap.common.RawRecord
+import org.broadinstitute.monster.dogaging.jadeschema.fragment.EolsDeath
+
+object DeathTransformations {
+
+  def mapDeathTransformations(rawRecord: RawRecord): EolsDeath = {
+    val deathLocation = rawRecord.getOptionalNumber("eol_death_location")
+    val deathWitness = rawRecord.fields.get("eol_who_present")
+    EolsDeath(
+      eolDeathLocation = deathLocation,
+      eolDeathLocationOtherDescription =
+        if (deathLocation.contains(98))
+          rawRecord.getOptional("eol_death_location_specify")
+        else None,
+      eolDeathWitness = rawRecord.getOptionalNumber("eol_anyone_present"),
+      eolDeathWitnessWhoYou = deathWitness.map(_.contains("1")),
+      eolDeathWitnessWhoFamily = deathWitness.map(_.contains("2")),
+      eolDeathWitnessWhoAcquaintance = deathWitness.map(_.contains("3")),
+      eolDeathWitnessWhoVet = deathWitness.map(_.contains("4")),
+      eolDeathWitnessWhoBoarder = deathWitness.map(_.contains("5")),
+      eolDeathWitnessWhoOther = deathWitness.map(_.contains("98")),
```

Comment:
The only comment I have is that I have been pulling out the "other" boolean fields as a variable first and then checking that variable when pulling the follow up text field but that's probably overkill for these one off instances so not sure if that should be a real nit

Using RC variable 'eol_who_present' as an example:
```
val deathWitness = rawRecord.fields.get("eol_who_present")
val deathWitnessOther = deathWitness.map(_.contains("98"))
EolsDeath(
    ...
    eolDeathWitnessWhoOther = deathWitnessOther,
    eolDeathWitnessWhoOtherDescription =
        if (deathWitnessOther.contains(true)) rawRecord.getOptional("eol_who_present_specify") else None,
    ...
)
```

Reply:
I like that suggestion, makes it consistent and more readable